src/status_im2/contexts/chat/composer/view.cljs

```diff
@@ -131,7 +132,7 @@
                        :background-y  background-y}
         props         (utils/init-props)
         state         (utils/init-state)]
-    [rn/view
+    [rn/view (when platform/ios? {:style {:z-index 1}})
```

Comment:
I think z-index is only supported on iOS, So we can just use it as it is without conditional.

Reply:
`z-index` is supported on iOS and Android. I want to update the z-index of iOS only here.